src/components/invitation/Invitation.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { Box, Grid, Typography } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+import theme from '../../styles/createTheme'
+
+const Invitation = () => {
+  const { classes } = useStyles()
+
+  return (
+    <Grid container spacing={3}>
+      <Grid item xs={12}>
+        <Typography variant="h1" className={classes.title}>
+          Invite
+        </Typography>
+        <Box
+          className={classes.imageWrapper}
+          component="img"
+          src="/img/invite/qr-invite.svg"
+        ></Box>
+        <Typography variant="body2" className={classes.description}>
+          Feel free to invite someone, who can be interesting in finding friends
+        </Typography>
+      </Grid>
+    </Grid>
+  )
+}
+
+export default Invitation
+
+const useStyles = makeStyles()(() => {
+  return {
```

Comment:
plz don't pull this `return`, it's not needed

Reply:
Done.